{{cookiecutter.project_name}}/pyproject.toml

```diff
@@ -27,14 +27,11 @@ flake8-bandit = "^4.1.1"
 flake8-broken-line = "^1.0.0"
 flake8-comprehensions = "^3.14.0"
 flake8-debugger = "^4.1.2"
-flake8-docstrings = "^1.7.0"
```

Comment:
pydocstyle wrapper. but pydocstyle is deprecated - https://github.com/PyCQA/pydocstyle/pull/658